### ISSUE TITLE
An ad tech company is using an XGBoost model to classify its clickstream data. The company’s Data Scientist is asked to explain how the model works to a group of non-technical colleagues. What is a simple explanation the Data Scientist can provide?

### ISSUE THREAD

**Issue by epassaro** [epassaro/aws-mls-c01#8]
### An ad tech company is using an XGBoost model to classify its clickstream data. The company’s Data Scientist is asked to explain how the model works to a group of non-technical colleagues. What is a simple explanation the Data Scientist can provide?

- [ ] XGBoost is an Extreme Gradient Boosting algorithm that is optimized for boosted decision trees
- [ ] XGBoost is a logistic regression algorithm to split each feature of the data and used for classification problem
- [ ] XGBoost is a robust, flexible, scalable algorithm that uses linear regression and used for regression problems
- [ ] XGBoost is an efficient and scalable neural network architecture

<details>
<summary><h3>Solución</h3></summary>

- [x] XGBoost is an Extreme Gradient Boosting algorithm that is optimized for boosted decision trees
- [ ] XGBoost is a logistic regression algorithm to split each feature of the data and used for classification problem
- [ ] XGBoost is a robust, flexible, scalable algorithm that uses linear regression and used for regression problems
- [ ] XGBoost is an efficient and scalable neural network architecture

</details>

<details>
<summary><h3>Razonamiento</h3></summary>
Esta pregunta me parece pésima. La respuesta correcta NO puede ser B, C o D ya que XGBoost no es un algoritmo de regresión logística, ni lineal, ni una arquitectura de red neuronal. Sin embargo, la parte de "non-technical"  distrae porque la respuesta A no se la darías a alguien que no tiene un background técnico.
</details>
